boto/awslambda/__init__.py

```diff
@@ -28,13 +30,10 @@ def regions():
     :rtype: list
     :return: A list of :class:`boto.regioninfo.RegionInfo`
     """
-    from boto.awslambda.layer1 import AWSLambdaConnection
```

Comment:
Was there a reason that we imported the connection object within the ``regions()`` function? Maybe @jamesls would know? Seems odd that this would be the pattern chosen without a particular reason.